optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -162,6 +162,7 @@ public SupplyManager getSupplyManager() {
 
     @Override
     public void setWorkingSolution(Solution_ workingSolution) {
+        Objects.requireNonNull(workingSolution);
```

Comment:
I like to do this:

![output](https://user-images.githubusercontent.com/673386/77458942-3dd9e680-6dff-11ea-8fb5-d772440d8d33.gif)

Reply:
Ah, it has a return type. Good to know, I thought it was `void`.